addons/html-embed/__tests__/html-embed.test.js

```diff
@@ -91,6 +91,24 @@ describe('Addons - HTML Embed', function() {
             target: { value: content }
           })
         })
+
+        it('keeps script tags when set to unsafe', function(done) {
```

Comment:
do you know if `onClick` and similar handlers can also create security holes?

Reply:
It definitely does. also <a href="javascript:">.

I'm starting to feel apathetic about it. I think the best approach would be to show warnings below the embed with a button to fix the problem. Doing it automatically for a user feels like a bad surprise from good intentions.

Also it's probably a breaking change, but one I think we should make. This at least fixes the immediate problem in a non-breaking way.